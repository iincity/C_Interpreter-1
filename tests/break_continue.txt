
int main()
{
	int i = 0;
	while (i < 20) {
		i += 1;
		if (i % 2)
			continue;
		if (i == 16)
			break;
		print i;
	}

	i = 0;
	while (i < 20) {
		i += 1;
		if (i % 2)
			continue;
		if (i == 16)
			break;
		print i;
		
		int j = 20;
		while (j) {
			j -= 1;
			if (j % 2)
				continue;
			if (j == 4)
				break;
			print j;
		}
	}

	for (int i=0, j=10; ; i+=1) {
		if (i % 2)
			continue;
		print i;
		if (i == j)
			break;
	}


	return 0;
}
