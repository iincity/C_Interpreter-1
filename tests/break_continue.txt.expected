2
4
6
8
10
12
14
2
18
16
14
12
10
8
6
4
18
16
14
12
10
8
6
6
18
16
14
12
10
8
6
8
18
16
14
12
10
8
6
10
18
16
14
12
10
8
6
12
18
16
14
12
10
8
6
14
18
16
14
12
10
8
6
0
2
4
6
8
10
